packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2653,8 +2653,64 @@ class SizedOverflowBox extends SingleChildRenderObjectWidget {
 /// painting anything, without making the child available for hit testing, and
 /// without taking any room in the parent.
 ///
+/// Difference in painting can be easily compared by the following sections of with and without Offstage.
+///
+/// ### Without Offstage
+///
+/// ```dart
+/// import 'package:flutter/material.dart';
+///
+/// void main() {
+///   runApp(MyApp());
+/// }
+///
+/// class MyApp extends StatelessWidget {
+///
+///   @override
+///   Widget build(BuildContext context) {
+///     return MaterialApp(
+///       home: Scaffold(
+///         body: Container(
+///           color: Colors.red,
+///           height: 120.0,
+///           width: 120.0,
+///         ),
+///       ),
+///     );
+///   }
+/// }
+/// ```
+///
+/// ### With Offstage
+///
+/// ```dart
```

Comment:
It would be better to have just one example that switches between the two states.

Reply:
I agree, But I have not added those to make it as simple as possible for beginners

Reply:
The example without the offstage seems too simple, though. In the interactive example it would be easier to see what offstage actually does by toggling between the two.